package.json

```diff
@@ -19,10 +19,11 @@
   },
   "homepage": "https://github.com/asyncapi/markdown-template#readme",
   "scripts": {
-    "test": "echo \"No test specified yet\"",
+    "test": "npm run test:generation",
     "lint": "echo \"No linter specified yet\"",
     "release": "semantic-release",
     "dev": "ag test/spec/asyncapi.yml ./ -o test/output --force-write --watch-template",
+    "test:generation": "ag https://raw.githubusercontent.com/asyncapi/generator/master/test/docs/dummy.yml ./ -o test/output --force-write",
```

Comment:
Do we want to tie the document to a specific version of the generator like the HTML template?

Reply:
yup, less risky to get false positive, if not under a release, then just choose a specific commit